Add reserved attributes to event migration doc (#14418)
DataDog/documentation: content/en/events/guides/migrating_to_new_events_features.md

Datadog automatically parses JSON-formatted events. When events are not JSON-formatted, they are parsed and enriched by chaining them sequentially through a processing pipeline. Processors extract meaningful information or attributes from semi-structured text to reuse as facets. Each event that comes through the pipelines is tested against every pipeline filter. If it matches a filter, then all the processors are applied sequentially before moving to the next pipeline.

## Reserved attributes

> Below is a list of reserved attributes that are automatically ingested with events:

Reviewer: ```suggestion
This list describes automatically ingested reserved attributes with events.
```

| Attribute | Description                                                                                                                                                                                                                                |
|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `host`    | The name of the originating host as defined in metrics. Datadog automatically retrieves corresponding host tags from the matching host in Datadog and applies them to your events. The Agent sets this value automatically.                          |

> | `source`  | This corresponds to the integration name: the technology from which the event originated. When it matches an integration name, Datadog automatically installs the corresponding parsers and facets. For example: `nginx`, `postgresql`, etc. |

Reviewer: ```suggestion
| `source`  | This corresponds to the integration name, or the technology from which the event originated. When it matches an integration name, Datadog automatically installs the corresponding parsers and facets. For example: `nginx`, `postgresql`, and more. |
```

> | `status`  | This corresponds to the level/severity of a event.      |

Reviewer: ```suggestion
| `status`  | This corresponds to the level or severity of an event.      |
```
